### ISSUE TITLE
Views drop after 20-30 minutes

### ISSUE THREAD

**Issue by Wodole8288** [KevinLage/YouTube-Livestream-Botter#88]
First of all thank you for developing this amazing bot.
I've made a couple of minor changes (loading the proxy from a list etc.), but couldn't get past one issue. Even with fresh private proxies, the views would go up to 1/2 and at best 2/3 the proxy amount and after 20-30 minutes the views would start dropping drastically.
Any help would be much appreciated.

**Comment by dhocin** [comment]
i dont know to using this github,,, token = sejNAOvMQEc("ID?\n")
cause when i using .exe i cant use as well i mean after 5minute it will drop views. so i want to another way and start using python but i dont know how to use it.
readme was not specific content. please help author. thankyou very much.
![Screenshot_4](https://user-images.githubusercontent.com/49015267/84562940-2cf79c00-ad82-11ea-9ae0-e8c76e962314.png)

**Comment by Wodole8288** [comment]
@dhocin The .exe files is just the .py file packed/compiled into the .exe format. If you're having troubles with the .exe file, don't expect anything to change by using the .py one.

**Comment by KevinLage** [comment]
> First of all thank you for developing this amazing bot.
> I've made a couple of minor changes (loading the proxy from a list etc.), but couldn't get past one issue. Even with fresh private proxies, the views would go up to 1/2 and at best 2/3 the proxy amount and after 20-30 minutes the views would start dropping drastically.
> Any help would be much appreciated.

youtube will prob. start blocking them

**Comment by arickryeland** [comment]
am getting this error whenever i tried to run it.. <blockquote class="imgur-embed-pub" lang="en" data-id="a/G9A6DrN" data-context="false" ><a href="//imgur.com/a/G9A6DrN"></a></blockquote><script async src="//s.imgur.com/min/embed.js" charset="utf-8"></script>

**Comment by KevinLage** [comment]
can u send some screenshots?

**Comment by arickryeland** [comment]
screenshot ` https://i.imgur.com/mdO9GErl.png`

**Comment by arickryeland** [comment]
<img width="526" alt="Screenshot 2020-06-15 at 5 43 56 AM" src="https://user-images.githubusercontent.com/66095818/84644497-3d685d80-aecd-11ea-8476-e78298237468.png">

**Comment by dhocin** [comment]
> @dhocin The .exe files is just the .py file packed/compiled into the .exe format. If you're having troubles with the .exe file, don't expect anything to change by using the .py one.

Yeah i know, even i want to another way, did means if i can using python ill get more option like i can changes proxy or anything, i hope so

**Comment by dhocin** [comment]
> can u send some screenshots?

Its alrwady kevin, but if i used exe antivirus will blocking that connection, then i want running in python, but let me show step by step. Thanks for the program and thanks for the answer

**Comment by arickryeland** [comment]
i run it with python but got that error..
<img width="526" alt="Screenshot 2020-06-15 at 5 43 56 AM" src="https://user-images.githubusercontent.com/66095818/84645611-d6e43f00-aece-11ea-95da-fc68cc5a6127.png">
